index.html

```diff
@@ -1,28 +1,29 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
   <title>try.hood.ie</title>
   <link rel="stylesheet" type="text/css" href="css/main.css">
   <link href='http://fonts.googleapis.com/css?family=Roboto+Slab:400,300' rel='stylesheet' type='text/css'>
 </head>
 <body>
-<img src="img/stripe.gif" id="stripe" />
-<header> 
+<img src="img/stripe.gif" id="stripe" alt=""/>
+<header role="banner"> 
     <h1>try.hood.ie</h1>
     <h2>create your app in a finger snap</h2>
-  </header>
-<div class="container">
+</header>
+<div role="main" class="container">
   <section class="current">
   		<form action="create">
         <div class="">
-          <input name="subdomain" required class="left" placeholder="your appname" />
+          <label for="subdomain"><b>Choose an application name</b></label><br />
+          <input id="subdomain" name="subdomain" required class="left" placeholder="your appname" />
           <span class="left label">.hoodieapp.com</span>
         </div>
   			<button type="submit" class="btn right hid">create me</button>
   		</form>
   </section>
 
-  <section class="desc">
+  <section class="desc" tabindex="-1" id="congratulations">
```

Comment:
I said briefly in the comment for the commit:

> In order for this content block to the focusable, I have added a tab index of '-1',
this denotes an element focusable by a script but not the user.

This is a poor explanation though!

In order to focus this section, we need to make the element tabbable, we can't `.focus()` on it without it being tabbable. A section does not have a default tab index and thus we have to add one. The tabindex of -1, like Lewis aluded to, means that we can use this as a focusable element programmatically (in this case, via JavaScript), but the user cannot tab to it itself.

I hope that helps a little more, but if not, Heydon Pickering wrote some great stuff about it here: http://www.sitepoint.com/learning-to-focus/